src/offchainapi/storage.py

```diff
@@ -3,10 +3,102 @@
 
 # The main storage interface.
 
+from hashlib import sha256
 import json
-from threading import RLock
-
+import contextvars
 from .utils import JSONFlag, JSONSerializable, get_unique_string
+import sqlite3
+
+class BasicStore:
```

Comment:
if i'm understanding it correctly, `BasicStore` is a reference implementation. If true, can we move it out of this file?